content/apps/creating-github-apps/authenticating-with-a-github-app/making-authenticated-api-requests-with-a-github-app-in-a-github-actions-workflow.md

```diff
@@ -60,3 +60,9 @@ jobs:
         run: |
           gh api octocat
 ```
+
+{% warning %}
+
+**Warning:** Automatic token protection prevents passing tokens generated with this method to subsequent jobs. By design, GitHub Runners are written to protect `GITHUB_TOKENS`.  As a consequence, they will set the values of GITHUB_OUTPUTS to the empty string if they contain tokens issued with this functionality.
+
+{% endwarning %}
```

Comment:
A couple of suggestions to:

- Use a note since we try to reserve warning callouts for potentially destructive actions
- Add a potential workaround
- Remove some text

```suggestion
{% note %}

**Note:** Due to automatic token protection, you cannot pass tokens generated with this method to downstream jobs. If you need to share the token with another job, you can store the token as an {% data variables.product.prodname_actions %} secret. For more information, see "[AUTOTITLE](/rest/actions/secrets#create-or-update-a-repository-secret)."

{% endnote %}
```

Reply:
Thank you for the review.  The potential workaround is not viable as it allows a token fetched by one workflow to be used by any other workflow in the repo which could be used to escalate privileges (as long as the attack was well timed).

Reply:
I'm happy to incorporate the comment elsewhere once we figure out the correct wording.

Reply:
In that case, would it be better to omit the workaround and just say this?

```
{% note %}

**Note:** Due to automatic token protection, you cannot pass tokens generated with this method to downstream jobs.

{% endnote %}
```